internal/pkg/aws/cloudwatch/cloudwatch.go

```diff
@@ -0,0 +1,140 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package cloudwatch provides a client to make API requests to Amazon CloudWatch Service.
+package cloudwatch
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/deploy/cloudformation/stack"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+)
+
+const (
+	compositeAlarmType = "Composite"
+	metricAlarmType    = "Metric"
+)
+
+type cwClient interface {
+	DescribeAlarms(input *cloudwatch.DescribeAlarmsInput) (*cloudwatch.DescribeAlarmsOutput, error)
+	ListTagsForResource(input *cloudwatch.ListTagsForResourceInput) (*cloudwatch.ListTagsForResourceOutput, error)
+}
+
+// CloudWatch wraps an Amazon CloudWatch client.
+type CloudWatch struct {
+	client cwClient
+}
+
+// App contains basic info for an application.
+type App struct {
+	AppName     string
+	EnvName     string
+	ProjectName string
+}
+
+// AlarmStatus contains CloudWatch alarm status.
+type AlarmStatus struct {
+	Arn          string
+	Name         string
+	Reason       string
+	Status       string
+	Type         string
+	UpdatedTimes int64
+}
+
+// New returns a CloudWatch struct configured against the input session.
+func New(s *session.Session) *CloudWatch {
+	return &CloudWatch{
+		client: cloudwatch.New(s),
+	}
+}
+
+// GetAlarms returns all CloudWatch alarms associated with the app.
+func (c *CloudWatch) GetAlarms(app App) ([]AlarmStatus, error) {
+	var alarmStatus []AlarmStatus
+	var err error
+	alarmResp := &cloudwatch.DescribeAlarmsOutput{}
+	for {
+		alarmResp, err = c.client.DescribeAlarms(&cloudwatch.DescribeAlarmsInput{
+			NextToken: alarmResp.NextToken,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("get CloudWatch alarms for app %s: %w", app.AppName, err)
```

Comment:
nit: since the request doens't contain anything specific to the app name.
```suggestion
			return nil, fmt.Errorf("describe CloudWatch alarms: %w", err)
```